assignments/array-methods.js

```diff
@@ -56,28 +56,55 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+ runners.forEach(runner => {
+     fullName.push(`${runner.first_name} ${runner.last_name}`);
+ });
+
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+allcaps = runners.map(runner => {
+ return runner.first_name.toLocaleUpperCase();
+});
+
 console.log(allCaps); 
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
 let largeShirts = [];
+largeShirts = runners.filter(runner => {
+   return runner.shirt_size.toLowerCase().indexOf(('L').toLowerCase()) > -1;
+});
```

Comment:
This is not returning just "L" size runners, but even XL 2XL etc..

Reply:
Oh Okay I thought It was supposed to return all larged sized shirts including XL 2XL. My mistake